site/docs/tasks/custom-taxonomy.md

```diff
@@ -51,3 +51,123 @@ helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait --set-file taxo
 ```
 The `--set-file` flag will pass in your custom `taxonomy.json` file to use for taxonomy validation in Fybrik.
 If this flag is not provided, Fybrik will use the default `taxonomy.json` file with no layers compiled into it. 
+
+
+After installing the fybrik helm chart, it is possible to upgrade Fybrik with an updated custom taxonomy file (`taxonomy.json`) with the following command:
```

Comment:
I would suggest replacing "After installing the fybrik helm chart" with "For an already deployed fybrik instance"

fybrik - with a small f